docs/testplan/platform/pmon-test-plan.md

```diff
@@ -74,49 +64,49 @@ CHASSIS_MODULE_INFO_SLOT_FIELD = 'slot'
 CHASSIS_MODULE_INFO_OPERSTATUS_FIELD = 'oper_status'
 ```
 
-### Test case # 4 – Data for ledd daemon
+### Test case # 3 – Data for ledd daemon
 #### Test objective
 Verify the expected data for ledd daemon (if not skip_ledd)
 #### Test steps
 * Verify the ledd running status
 
-### Test case # 5 – Data for pcied daemon
+### Test case # 4 – Data for pcied daemon
 #### Test objective
 Verify the expected data for pcied daemon (if not skip_pcied)
 #### Test steps
 * Verify the pcie config file "/usr/share/sonic/platform/pcie.yaml" exists
 * Verify the "PCIE_DEVICES" has "status" field set to "SUCCESS" 
 
-### Test case # 6 – Data for psud daemon
+### Test case # 5 – Data for psud daemon
 #### Test objective
 Verify the expected data for psud daemon (if not skip_psud)
 #### Test steps
 * Verify the 'PSU_INFO|PSU {}' is/are available
 * Verify the information data for each 'PSU {}' is expected to compare "/usr/share/sonic/platform/platform.json"
 
-### Test case # 7 – Data for syseepromd daemon
+### Test case # 6 – Data for syseepromd daemon
 #### Test objective
 Verify the expected data for syseepromd daemon (if not skip_syseepromd)
 #### Test steps
 * Verify the 'EEPROM_INFO|State' in state_db has the 'Initialized' field set to '1'
 * Verify the 'EEPROM_INFO|*' has valid pairs of key and value
-   * Q) how to verify the data is valid value or if we can skip the check for the value?
+   * Verify the data to check with the output of the platform api 
 
-### Test case # 8 – Data for thermalctld daemon
+### Test case # 7 – Data for thermalctld daemon
 #### Test objective
 Verify the expected data for thermalctld daemon (if not skip_thermalctld)
 #### Test steps
 * Verify the policy file "/usr/share/sonic/platform/thermal_policy.json" exists
 * Verify the 'FAN_INFO|{}' and 'TEMPERATURE_INFO|{}' available
-   * Q) how to verify the data is valid value ? log check?
+   * Verify the data to check with the output of the platform api 
 
-### Test case # 9 – Data for xcvrd daemon
+### Test case # 8 – Data for xcvrd daemon
 #### Test objective
 Verify the expected data for xcvrd daemon (if not skip_xcvrd)
 #### Test steps
 * Verify the 'TRANSCEIVER_INFO', 'TRANSCEIVER_DOM_SENSOR' and 'TRANSCEIVER_STATUS' state_db table has data
 * Verify the information data for each 'TRANSCEIVER_INFO' is expected to compare "/usr/share/sonic/platform/platform.json"
-   * Q) how to verify the data is valid value? log check?
+   * Verify the data to check with the output of the platform api 
```

Comment:
Rephrase: `For each entry in State DB, gather the data by calling the appropriate Platform API call directly and ensure they match (for variable data, like temperatures and voltages, ensure they are within a specified tolerance`